active/radiobuttons/RadioButtons_Spec.md

```diff
@@ -0,0 +1,288 @@
+    static Windows.UI.Xaml.DependencyProperty SelectedItemProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaximumColumnsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
+}
+
+}
+
+```
+
+## RadioButtons Primitives
+
+``` c++
+namespace MU_XCP_NAMESPACE
+{
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+[default_interface]
+unsealed runtimeclass RadioButtonsElementFactory : MU_XC_NAMESPACE.ElementFactory
+{
+    RadioButtonsElementFactory();
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+[default_interface]
+unsealed runtimeclass ColumnMajorUniformToLargestGridLayout : MU_XC_NAMESPACE.NonVirtualizingLayout
+{
+    ColumnMajorUniformToLargestGridLayout();
+
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Int32 MaximumColumns{ get; set; };
+    static Windows.UI.Xaml.DependencyProperty MaximumColumnsProperty{ get; };
+
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Int32 ColumnSpacing{ get; set; };
+    static Windows.UI.Xaml.DependencyProperty ColumnSpacingProperty{ get; };
+
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Int32 RowSpacing{ get; set; };
+    static Windows.UI.Xaml.DependencyProperty RowSpacingProperty{ get; };
+}
+
+}
+
+```
+
+## RadioButtons.xaml
+
+``` xml
+<Style TargetType="local:RadioButtons" BasedOn="{StaticResource DefaultRadioButtonsStyle}" />
+
+<Style x:Key="DefaultRadioButtonsStyle" TargetType="local:RadioButtons">
+    <Setter Property="IsTabStop" Value="False" />
+    <Setter Property="TabNavigation" Value="Once" />
+    <Setter Property="Template">
+        <Setter.Value>
+            <ControlTemplate TargetType="local:RadioButtons">
+                <StackPanel>
+                    <ContentPresenter Content="{TemplateBinding Header}"/>
+                    <local:ItemsRepeater x:Name="InnerRepeater" ItemTemplate="{TemplateBinding ItemTemplate}">
+                        <local:ItemsRepeater.Layout>
+                            <primitives:ColumnMajorUniformToLargestGridLayout ColumnSpacing="7" RowSpacing="3"/>
+                        </local:ItemsRepeater.Layout>
+                    </local:ItemsRepeater>
+                </StackPanel>
+                </ControlTemplate>
+        </Setter.Value>
+    </Setter>
+</Style>
+```
+
+# API Notes
+
+| Name | Description |
+|:-:|:--|
+| Header | Places a label object above the group of RadioButton elements and is exposed to the UIA tree via the DataContext property on the RadioButtons control's ContentPresenter. |
+| SelectedItem | Gets the content of the selected RadioButton item. Selection is denoted by that item's RadioButton icon being checked. |
+| SelectedIndex | Gets or sets the selected RadioButton item by index. Selection is denoted by that item's RadioButton icon being checked.|
+| Items | Gets or sets an object source used to generate the content of the ItemsControl. All items placed within the RadioButtons control will get the content of a generated RadioButton. |
```

Comment:
This description sounds like ItemsSource, not Items.

Reply:
Attempted to redefine it. Please double check for clarity @StephenLPeters 😄